examples/script.py

```diff
@@ -0,0 +1,572 @@
+#APIn  Method of Linking
+import os
+from cffi import FFI
+import ctypes
+
+from sys import exit, platform
+
+ffi = FFI()
+cdef_from_file = None   
+header = '/mnt/host/tmp/workspace/cocosdk/cocodevicesdk/dist/c/0.65.2/include/cocodevicesdk/coco_device_api.h'
+
+#Accessing C header file try and exceptions
+try:
+    with open(header, 'r') as libtestcffi_header:
+        cdef_from_file = libtestcffi_header.read()
+except FileNotFoundError:
+    print('Unable to find "%s"' % header)
+    exit(-1)
+except IOError:
+    print('Unable to open "%s"' % header)
+    exit(-1)
+finally:
+    if cdef_from_file == '':
+        print('File "s" is empty' % header)
+        exit(-1)
+
+#ffi.cdef(cdef_from_file)
```

Comment:
delete this. IF this is not being used

Reply:
This will be used once there is a header file containing all of the total declarations needed for the entire library, I will comment out for now